### ISSUE TITLE
All page content should be contained by landmarks

### ISSUE THREAD

**Issue by vh8154** [pulibrary/DSS#473]
### Steps to recreate
1. Go to https://dss.princeton.edu/regions
2. Log in
3. Visit [this page](https://dss.princeton.edu/regions)
4. click "edit" for one of the rows
5.  click "Update Region".
6. Run axe DevTools on this page.

### URL
https://dss.princeton.edu/regions

### Issues
All page content should be contained by landmarks

### Severity
Moderate

### WCAG Success Criterion
[WCAG 1.3.6: Identify Purpose](https://www.w3.org/WAI/WCAG22/Understanding/identify-purpose)
[WCAG 2.4.1: Bypass Blocks](https://www.w3.org/WAI/WCAG22/Understanding/bypass-blocks)

### WCAG Conformance level (A, AA, AAA)
AAA

### User impact
Users who utilize screen-readers may find it more difficult to bypass large sections of content or identify the purpose of the content.

### Recommended fix
To solve this problem, you need to fix the following:
- Some page content is not contained by landmarks

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

**Comment by vh8154** [comment]
Closed by #476 
The upgrade to blacklight fixed this issue.
